code/modules/mob/living/carbon/human/death.dm

```diff
@@ -41,9 +41,14 @@
 
 	for(var/obj/Overlays/O in obj_overlays)
 		returnToPool(O)
-	
+
 	obj_overlays = null
 
+	var/datum/species/s = species
```

Comment:
Oh, this is human destroy, I can just null it.

Originally I casted it because species can be either text or the species datum, depending on where along the line of setting its species it is.